app/views/RegisterView.tsx

```diff
@@ -307,18 +306,16 @@ class RegisterView extends React.Component<IProps, any> {
 						style={styles.registerButton}
 					/>
 
-					<UGCRules />
-
 					{showLoginButton ? (
 						<View style={styles.bottomContainer}>
 							<Text style={[styles.bottomContainerText, { color: themes[theme].fontSecondaryInfo }]}>
```

Comment:
Not matching Figma

Reply:
@yash-rajpal did you see this?

Reply:
I thought it was related to inside this component 🤔 

Let me see.

Reply:
Updated a few styles, please check. 😬

Reply:
Also here - 
![image](https://github.com/user-attachments/assets/4b3b2538-e4a0-47c4-84dd-ce5e1adbb6bf)
Not updating `App version` to `Version` as in styles as this change was recently made [here](https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/5846) and the designs are older than that.